### ISSUE TITLE
Allow passing in a string as txHash to provider.txStatus

### ISSUE THREAD

**Issue by barbaraliau** [near/near-api-js#218]
Other Provider methods take in a `string` or `number`. For consistency and improved user experience, consider updating `txHash` to accept a `string` as well.

If this is not feasible at the time, updating the error message to inform the user that a `Uint8Array` is expected is more helpful than the current error message: `Error: [-32700] Parse error: incorrect length for hash`

```ts
export declare abstract class Provider {
    abstract getNetwork(): Promise<Network>;
    abstract status(): Promise<NodeStatusResult>;
    abstract sendTransaction(signedTransaction: SignedTransaction): Promise<FinalExecutionOutcome>;
    abstract txStatus(txHash: Uint8Array, accountId: string): Promise<FinalExecutionOutcome>;
    abstract query(path: string, data: string): Promise<any>;
    abstract block(blockId: BlockId): Promise<BlockResult>;
    abstract chunk(chunkId: ChunkId): Promise<ChunkResult>;
}
```

https://github.com/nearprotocol/nearlib/blob/master/lib/providers/provider.d.ts#L120

**Comment by janedegtiareva** [comment]
Thank you for reporting!
This is similar to issue https://github.com/near/near-api-js/issues/121

**Comment by mehtaphysical** [comment]
Fixed in #561
